funflow/src/Control/FunFlow/External.hs

```diff
@@ -19,11 +20,28 @@ import           GHC.Generics                    (Generic)
 import           Path
 import           System.Posix.Types              (CGid, CUid)
 
+-- | Set of items which may be treated as an input path to an external task.
+data InputPath
+    -- ^ An item in the content store.
```

Comment:
Sorry, didn't see this before, I think this breaks haddock.

Reply:
https://www.haskell.org/haddock/doc/html/ch03s02.html says this is fine - I think it's only GADTs where we can't document constructors.